galaxy/Dockerfile

```diff
@@ -81,13 +83,14 @@ ADD ./htpasswd /etc/nginx/htpasswd
 ADD roles/ /tmp/ansible/roles
 ADD provision.yml /tmp/ansible/provision.yml
 RUN ansible-playbook /tmp/ansible/provision.yml \
+    --extra-vars galaxy_venv_dir=$GALAXY_VIRTUAL_ENV \
```

Comment:
No?

Galaxy will consume the environment variable but I don't think the ansible role will right?